modules/ROOT/pages/values-and-types/converting-data.adoc

```diff
@@ -0,0 +1,134 @@
+This section will provide an overview of those functions, as well examples of how to use them in practice. 
+
+== Functions for converting data values
+
+The following functions are available for converting data values:
+
+[options="header", cols="m,2a"]
+|===
+| Function | Description
+
+| toBoolean() | Converts a string, integer, or boolean value to a boolean value.
+
+| toBooleanList() | converts a list of values and returns a list of boolean values.
+If any values are not convertible to boolean they will be `null` in the list returned.
+
+| toBooleanOrNull() | Converts a string, integer or boolean value to a boolean value.
+For any other input value, `null` will be returned.
+
+| toFloat() | Converts an integer, floating point, or a string value to a floating point number value.
+Otherwise `null` is returned.
+
+| toFloatList() | Converts a list of values and returns a list of floating point values.
+If any values are not convertible to floating point they will be `null` in the list returned.
+
+| toFloatOrNull() | Converts an integer, floating point, or a string value to a floating point number.
+For any other input value, `null` will be returned.
+
+| toInteger() | Converts a boolean, integer, floating point or a string value to an integer value.
+
+| toIntegerList() | Converts a list of values and returns a list of integer values. If any values are not convertible to integer they will be `null` in the list returned.
+
+| toIntegertOrNull() | Converts a boolean, integer, floating point or a string value to an integer value.
+For any other input value, `null` will be returned.
+
+| toString() |  Converts an integer, float, boolean, string, point, duration, date, time, localtime, localdatetime, or datetime value to a string value.
+
+| toStringList() | Converts a list of values and returns a list of string values.
+If any values are not convertible to string they will be `null` in the list returned.
+
+| toStringOrNull() | Converts an integer, float, boolean, string, point, duration, date, time, localtime, localdatetime, or datetime value to a string.
+For any other input value, `null` will be returned.
+|===
+
+More information about these, and many other functions, can be found in the section on xref:functions/index.adoc[].
+
+[[converting-data-values-examples]]
+== Examples
+
+The below graph is used for the examples below:
+
+image::values_and_types_converting_data_graph.svg[]
+
+To recreate it, run the following query against an empty Neo4j database:
+
+[source, cypher, role=noheader,test-setup]
+----
+CREATE (keanu:Person {name:'Keanu Reeves', age: 58, active:true}),
+       (carrieAnne:Person  {name:'Carrie-Anne Moss', age: 55, active:true}),
+       (keanu)-[r:KNOWS {since:1999}]->(carrieAnne)
+----
+
+[[converting-data-values-examples-returning-converted-values]]
+=== Returning converted values
+
+In the below query, the function `toFloat` is used to convert two string values.
```

Comment:
Would it be useful to show that toFloat will error if passed say a Date and toFloatOrNull still returns null? I found it not super obvious myself that that was the difference between them

Reply:
I think that would make sense - I will add an example to that effect tomorrow.